MotorRandomWee/MotorRandomWee.ino

```diff
@@ -59,6 +60,7 @@ void setMotorSpeedRandom(int potValLeft, int potValRight){
   count++;
 }
 
+//SET MOTOR SPEED WITH CONSOLE INPUT/INT INPUT
 void setMotorSpeed(int rightSpeed){
   const double MAXSPEED = 1300, MINSPEED = 900;
   int rightMotorSpeed = rightSpeed * (MAXSPEED - MINSPEED) + MINSPEED; 
```

Comment:
There should also be some protection in case the PID algorithm sends more than the allotted values, something like 
```suggestion
  if(rightMotorSpeed > 100) {rightMotorSpeed = 100;}
```
and also for when it's 0 as well